app/models/task.rb

```diff
@@ -0,0 +1,30 @@
+class Task < ApplicationRecord
+	belongs_to :project
+
+	validates :title, presence: true
+	validates :deadline_at, presence: true
+	validates :project, presence: true
+
+	validate :deadline_at_after_created_at
+	# validates_with MyValidator
+
+	def deadline_at_after_created_at
+		return if (created_at || Time.current) < deadline_at
+
+		errors.add(:deadline_at, "must be after the date of")
```

Comment:
Poor error message. By error, it will not be very clear what the problem is.

Reply:
okay, I will give correct error messages